swiss-ng-api.yaml

```diff
@@ -6015,13 +6023,15 @@ components:
           $ref: "#/components/schemas/chargeBearer"
         remittanceInformationUnstructured:
           $ref: "#/components/schemas/remittanceInformationUnstructured"
-        remittanceInformationUnstructuredArray:
-          $ref: "#/components/schemas/remittanceInformationUnstructuredArray"
+#       remittanceInformationUnstructuredArray:
+#         $ref: "#/components/schemas/remittanceInformationUnstructuredArray"
         remittanceInformationStructured: # N.A. for all, but can be used by ASPSP if needed
           $ref: "#/components/schemas/remittanceInformationStructured"
-        requestedExecutionDate: # N.A. for all, but can be used by ASPSP if needed
-          type: string
-          format: date
+#       remittanceInformationStructuredArray: # N.A. for all, but can be used by ASPSP if needed
+#         $ref: "#/components/schemas/remittanceInformationStructuredArray"
+#       requestedExecutionDate: # N.A. for all, but can be used by ASPSP if needed
+#         type: string
+#         format: date
```

Comment:
The **requestedExecutionDate** is actually mandatory for all. but the **requestedExecutionTime** is not and should be commented.

Reply:
You're right, we'll be corrected.